resources/views/settings/delete-account.blade.php

```diff
@@ -6,26 +6,94 @@
+    <p>{{ __('If you delete your account:') }}</p>
+    @if ($user->context === 'individual')
+        <ul>
+            <li>
+                <p>{{ __('you will no longer be able to access information about the [number] engagements you are participating in') }}
+                </p>
+            </li>
+            <li>
+                <p>{{ __('you will no longer be able to access information about the [number] projects you are contracted for') }}
+                </p>
+            </li>
+            <li>
+                <p>{{ __('you will no longer be able to access your [number] training certificates') }}</p>
+            </li>
+            <li>
+                <p>{{ __('your public profile will be removed from the platform') }}</p>
+            </li>
+            <li>
+                <p>{{ __('you will no longer be matched to any projects and engagements') }}</p>
+            </li>
+        </ul>
+    @elseif($user->context === 'organization')
+        <ul>
+            <li>
+                <p>{{ __('you will no longer be able to access information about the [number] projects you are contracted for') }}
+                </p>
+            </li>
+            <li>
+                <p>{{ __('you will no longer be able to access information about the [number] projects you are participating in') }}
+                </p>
+            </li>
+            <li>
+                <p>{{ __('you will no longer be able to manage the [number] projects you are running') }}</p>
+            </li>
+            <li>
+                <p>{{ __('you will no longer be able to access your [number] training certificates') }}</p>
+            </li>
+            <li>
+                <p>{{ __('your public profile will be removed from the platform') }}</p>
+            </li>
+        </ul>
+    @elseif($user->context === 'regulated-organization')
+        <ul>
+            <li>
+                <p>{{ __('you will no longer be able to manage the [number] projects you are running') }}</p>
+            </li>
+            <li>
+                <p>{{ __('you will no longer be able to access your [number] training certificates') }}</p>
+            </li>
+        </ul>
+    @endif
 
-    <p>{{ __('hearth::user.delete_account_intro') }}</p>
+    @if (!($user->isOnlyAdministratorOfOrganization() || $user->isOnlyAdministratorOfRegulatedOrganization()))
+        <div role="alert">
+            <x-hearth-alert type="warning">
+                {{ __('Once you delete your account, you will not be able to recover it.') }}
+            </x-hearth-alert>
+        </div>
+    @endif
 
-    <form action="{{ localized_route('users.destroy') }}" method="post" novalidate>
+    <!-- Form Validation Errors -->
+    @include('partials.validation-errors')
+    <form class="@if ($user->isOnlyAdministratorOfOrganization() || $user->isOnlyAdministratorOfRegulatedOrganization()) disabled @endif" action="{{ localized_route('users.destroy') }}"
```

Comment:
@greatislander I know you suggested me to update class name in this manner over ternary operator, but this method adds extra white spaces in the class name; the class name becomes " disabled ", so the style in _form.css is not applying as expected.

Reply:
oh, I see when I just hard coded class name to be "disabled", the style is till not applied.